changelog

```diff
@@ -1,3 +1,50 @@
+Version 0.10.12
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@reticent-rem, @TheGiraffe3, @tibetiroka, @TomGoodIdea, @warp-core)
+      * The job "Paradise Job: Wilderness Retreat" will no longer select a station as its destination. (@TheGiraffe3)
+      * The "Dangerous Games" missions will now only offer from planets within 100 hyperspace links of their destinations, instead of potentially in disconnected regions. (@warp-core)
+      * Fixed some log entry actions for Tim the stowaway that were malformed. (@warp-core)
+      * Added a missing passenger to the "Human to Hai Space - No Contact" mission. (@TheGiraffe3)
+      * Changed the description of the Tractor Beam to refer to "attractor" rather than "repulsor" technology. (@mOctave)
+      * The locations for distressed ships in Remnant rescue jobs one through three are now restricted to systems connected by hyperlane or wormhole to Arculus, so the objective ships can always make it back home. (@warp-core)
+    * Engine bugs:
+      * Fixed an issue where player escorts could start jumping around independently of the flagship. (@warp-core)
+      * Tooltip boxes will no longer be drawn for items without tooltip text in ship and outfit info displays. (@warp-core)
+      * When determining if there are sufficient resources to fire a weapon, only the values of resources firing the weapon actually consumes will be considered. (@warp-core)
+      * Exceptions thrown by std::copy in Files::Copy will now be caught, instead of crashing the game. (@TomGoodIdea)
+  * Game content:
+    * New content:
+      * Added launch sounds and angles to bays on Avgi ships. (@Saugia)
+    * Mission changes:
+    * Balance:
+      * Buffed the health and mobility of the Bastion and Bulwark. (@ziproot)
+        * Bastion:
+          * Shields: 7500 -> 8600
+          * Hull: 4700 -> 5300
+          * Mass: 1210 -> 1100
+          * Drag: 16.5 -> 15.5
+          * Heat dissipation: 0.32 -> 0.34
+        * Bulwark:
+          * Shields: 7000 -> 8000
+          * Hull: 5600 -> 6500
+          * Mass: 1722 -> 1582
+          * Drag: 21 -> 20.1
+          * Heat dissipation: 0.27 -> 0.28
+      * Korath ships raiding human space now have the "frugal" personality. (@Quantumshark)
+    * Other:
+      * Slightly changed some wording in the "Saving Ildico" mission chain. (@Arachi-Lover)
+      * Avgi map tweaks? What did he change? (@Azure3141)
+      * Expanded the conversation in "Remnant: Cognizance 29" to account for the total destruction of the Korath fleet waiting in the system in the previous mission. (@warp-core)
+  * Game mechanics:
+  * User interface:
+    * Added a tooltip for the "automaton" attribute. (@TomGoodIdea)
+    * The unexplored system rings in the map now use a named color, defined in the interfaces file. (@TomGoodIdea)
+    * Added a tooltip for the "flotsam chance" attribute. (@Azure3141)
+  * Under the hood:
+    * Added some missing #includes made necessary by GCC 15. (@Koranir)
```

Comment:
I've also had this problem on GCC 14.2.1, not sure which version has introduced it.

Reply:
```suggestion
    * Added some missing #includes required by some C++ STL implementations. (@Koranir)
```